install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -158,6 +158,8 @@ spec:
             httpHeaders:
             - name: "brief"
               value: "true"
+            - name: "require-k8s-connectivity"
```

Comment:
> The old behavior can be turned back on by setting the header 'require-k8s-connectivity: "true"' in the livenessProbe section of the cilium daemonset.

but there is no easy way to switch it through helm I think? I guess ideally it would have helm value and we could mention that in upgrade notes.

Reply:
Good point! Added `livenessProbe.requireK8sConnectivity` helm option.